ios/Inji.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -34,13 +34,13 @@
       "location" : "https://github.com/mosip/inji-openid4vp-ios-swift",
       "state" : {
         "branch" : "release-0.1.x",
-        "revision" : "9e7677440e561924289e4041522713d9dfe9dbd7"
+        "revision" : "2f12d1c12b87caa4aeb6894efd66d33893f1ebd8"
       }
     },
     {
       "identity" : "inji-vci-client-ios-swift",
       "kind" : "remoteSourceControl",
-      "location" : "https://github.com/mosip/inji-vci-client-ios-swift",
+      "location" : "https://github.com/mosip/inji-vci-client-ios-swift/",
```

Comment:
why added `/`

Reply:
This file gets auto generated by xcode..No manual addition has been done